py/torch_tensorrt/dynamo/conversion/converter_utils.py

```diff
@@ -15,9 +15,11 @@
     ConverterRegistry,
     DynamoConverterImplSignature,
 )
-from torch_tensorrt.fx.converters.converter_utils import (
+from torch_tensorrt.fx.converters.converter_utils import (  # noqa: F401
     broadcast,
     get_axes_for_reduce_op,
+    prepend_ones,
+    set_layer_name,
```

Comment:
Copy the source of these functions directly into this file here. 
1) We should not be using utilities from `torch_tensorrt.fx.converters.converter_utils` but rather have them in `torch_tensorrt.dynamo.converters.converter_utils`
2) Also, add these types to our dynamo types `from torch_tensorrt.fx.types import TRTDataType, TRTTensor` 
One of the side goals of adding DS support for converters is to move away from using legacy utilities (`torch_tensorrt.fx` in this case).